src/main/scala/djinni/CppGenerator.scala

```diff
@@ -217,37 +217,38 @@ class CppGenerator(spec: Spec) extends Generator(spec) {
                   )
                   .mkString(",")
 
-            w.wl(s"static void to_json(nlohmann::json& j, ${ident.name} e)")
```

Comment:
:eyes: scalafmt - nothing was changed other than the addition of the `inline` specifier.